frontend/components/cards/ModuleUnionCard.tsx

```diff
@@ -51,41 +52,61 @@ const ModuleUnionCard: FC<ModuleUnionCardProps> = ({
   switch (module.model) {
     case "images.image":
       // return <ImageCard image={module} {...childProps} />;
-      content = <HTMLEllipsis unsafeHTML={module.captionHtml} maxLine="3" basedOn="words" />;
+      content = (
+        <HTMLEllipsis
+          unsafeHTML={applyHTMLFilter(module.captionHtml)}
+          maxLine="3"
+          basedOn="words"
+        />
+      );
       break;
     case "propositions.occurrence":
       content = (highlightSnippet && <HighlightEllipsis unsafeHTML={highlightSnippet} />) || (
-        <HTMLEllipsis unsafeHTML={module.truncatedElaboration} maxLine="2" basedOn="words" />
+        <HTMLEllipsis
+          unsafeHTML={applyHTMLFilter(module.truncatedElaboration)}
+          maxLine="2"
+          basedOn="words"
+        />
       );
       break;
     case "propositions.proposition":
       content = (highlightSnippet && <HighlightEllipsis unsafeHTML={highlightSnippet} />) || (
-        <HTMLEllipsis unsafeHTML={module.truncatedElaboration} maxLine="2" basedOn="words" />
+        <HTMLEllipsis
+          unsafeHTML={applyHTMLFilter(module.elaboration)}
+          maxLine="2"
+          basedOn="words"
+        />
       );
       break;
     case "quotes.quote": {
       content = (
-        <blockquote className="blockquote">
-          <div>
-            {(highlightSnippet && <HighlightEllipsis unsafeHTML={highlightSnippet} />) ??
-              (module.bite && (
-                <HTMLEllipsis unsafeHTML={module.bite} maxLine="3" basedOn="words" />
-              ))}
-          </div>
-        </blockquote>
+        //<blockquote className="blockquote">
```

Comment:
IMO if we don't need this `blockquote` element anymore, we should probably just delete the lines.